### PR TITLE
3377 isogenity audits

### PR DESCRIPTION
**Pull request by cricketsloan**
This is a total redo of the replication type.  The audits are being removed and a facet for isogenity is added.

### CODE REVIEW

**Comment by lrowe** [comment]
Travis shows this test failing on the branch: test_audit_experiment_model_organism_mismatched_age

**Comment by cricketsloan** [comment]
I will check it out.


On Nov 18, 2015, at 5:14 PM, Laurence Rowe <notifications@github.com<mailto:notifications@github.com>> wrote:


Travis shows this test failing on the branch: test_audit_experiment_model_organism_mismatched_age

**Comment by strattan** [comment]
Friends don’t let friends say “isogenity” https://github.com/ENCODE-DCC/encoded/pull/1024.

I don’t know if it’s a word or what it might mean.  It sounds vaguely anatomical.

Stick with isogenic, isogenicity  Avoid isogeneic/isogeneity: these have specific meanings in immunology that differ from what we’re trying to say about replicates.  So not audit_experiment_isogeneity

Seth
J. Seth Strattan, PhD --- jseth@stanford.edu<mailto:jseth@stanford.edu> --- www.encodeproject.org<http://www.encodeproject.org> --- 650-725-4735

On Nov 19, 2015, at 8:38 AM, Cricket Alicia Sloan <notifications@github.com<mailto:notifications@github.com>> wrote:


I will check it out.


On Nov 18, 2015, at 5:14 PM, Laurence Rowe <notifications@github.com<mailto:notifications@github.com><mailto:notifications@github.com>> wrote:


Travis shows this test failing on the branch: test_audit_experiment_model_organism_mismatched_age

**Comment by cricketsloan** [comment]
Thanks, Seth.  I have changed my ways and renamed to audit_experiment_isogenic_state.  I will refrain from using isogeneity and any misspellings of that word to avoid the confusion with the immunology definition.

Cricket

On Nov 19, 2015, at 8:43 AM, J. Seth Strattan <notifications@github.com<mailto:notifications@github.com>> wrote:

Friends don’t let friends say “isogenity” https://github.com/ENCODE-DCC/encoded/pull/1024.

I don’t know if it’s a word or what it might mean. It sounds vaguely anatomical.

Stick with isogenic, isogenicity Avoid isogeneic/isogeneity: these have specific meanings in immunology that differ from what we’re trying to say about replicates. So not audit_experiment_isogeneity

Seth
J. Seth Strattan, PhD --- jseth@stanford.edu<mailto:jseth@stanford.edu><mailto:jseth@stanford.edu> --- www.encodeproject.org<http://www.encodeproject.org><http://www.encodeproject.org> --- 650-725-4735

On Nov 19, 2015, at 8:38 AM, Cricket Alicia Sloan <notifications@github.com<mailto:notifications@github.com><mailto:notifications@github.com>> wrote:


I will check it out.


On Nov 18, 2015, at 5:14 PM, Laurence Rowe <notifications@github.com<mailto:notifications@github.com><mailto:notifications@github.com><mailto:notifications@github.com>> wrote:


Travis shows this test failing on the branch: test_audit_experiment_model_organism_mismatched_age

File: src/encoded/types/experiment.py [hunk 1]
```diff
@@ -273,29 +263,6 @@ def replication_type(self, request, replicates=None):
...
-                'assay_term_name',
-                'target.investigated_as',
-                'month_released',
-                'files.file_type',
-            ],
-            'group_by': 'assay_term_name',
-            'label': 'Assay',
-        },
-    }
-
```
Comment (lrowe) [inline src/encoded/types/experiment.py]: Looks like you have a merge conflict here. This is part of the matrix branch that got merged recently.

**Comment by lrowe** [comment]
Superseded by the 3377-iso-audits-idan branch.

**Pull request closed without merging**